src/DynamoCoreWpf/UI/Prompts/ChangeScaleFactorPrompt.xaml.cs

```diff
@@ -12,16 +14,43 @@ namespace Dynamo.Prompts
     /// </summary>
     public partial class ChangeScaleFactorPrompt : Window
     {
+        public enum Size
+        {
+            Small = 0,
+            Medium = 1,
```

Comment:
Since the number follow the enum is actually the grid column number as they appear in the ui, should I follow this order or just put medium as the first one?